{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/users/tests/test_views.py

```diff
@@ -1,8 +1,10 @@
 import pytest
 from django.conf import settings
 from django.test import RequestFactory
+from django.urls import reverse
 
-from {{ cookiecutter.project_slug }}.users.views import UserRedirectView, UserUpdateView
+from .factories import UserFactory
```

Comment:
```suggestion
from {{ cookiecutter.project_slug }}.users.tests.factories import UserFactory
```

Reply:
Thanks for your review. Does this test resolve https://github.com/pydanny/cookiecutter-django/issues/1020?

Reply:
I think it is, yes. This a fairly old issue so it might have been resolved already.

Reply:
Actually, it might be happening when the URL is accessed with lowercase?

```
reverse(viewname="users:detail", kwargs={"username": user.username.lower()})
```